integration/spark/app/integrations/container/pysparkPhysicalLocationComplete.json

```diff
@@ -0,0 +1,33 @@
+{
+  "eventType" : "COMPLETE",
+  "job" : {
+    "namespace" : "physicalLocation"
+  },
+  "outputs" : [
+    {
+      "namespace": "hive://metastore",
+      "name": "default.test",
+      "facets": {
+        "dataSource": {
+          "name": "hive://metastore",
+          "uri": "hive://metastore"
+        },
+        "schema": {
+          "fields": [
+            {
+              "name": "key",
+              "type": "integer"
+            },
+            {
+              "name": "value",
+              "type": "string"
+            }
+          ]
+        },
+        "storage": {
+          "physicalLocation": "/tmp/warehouse/test"
```

Comment:
If you're reusing facet to set one field, and not set any other fields set by this facet, it seems that the field itself is a good candidate to be in a separate facet.

Reply:
On the other hand, `hive` should probably be a valid value for `storageLayer`